devenv/start_here.sh

```diff
@@ -17,39 +12,73 @@
 export WORKON_HOME=~/.virtualenvs
```

Comment:
This does not seem to be used anymore. One can use `cdvirtualenv` or `$VIRTUAL_ENV` to go in the virtual environment directory.